runtime/queries/metricsview.go

```diff
@@ -340,6 +340,22 @@ func identifierIsUnnest(mv *runtimev1.MetricsViewSpec, expr *runtimev1.Expressio
 	return false
 }
 
+func dimensionSelect(mv *runtimev1.MetricsViewSpec, dim *runtimev1.MetricsViewSpec_DimensionV2, dialect drivers.Dialect) (string, string) {
```

Comment:
Use named return variables to indicate what it returns  (or add a docstring about it)

Reply:
Used named variables.